README.md

```diff
@@ -31,3 +31,34 @@ Contributors
 Contributing
 ------------
 Please read more about [Github Flow](https://guides.github.com/introduction/flow/).
+
+Getting started
+---------------
+
+In order to customize docker skeleton:
+
+1. open `.project_name` file and set project name
+2. open `docker.sh` script and set below variables
+
+```
+export PROJECT_WEB_DIR=${PROJECT_WEB_DIR:="web"} # change 'web' if it is required
+export PROJECT_INDEX_FILE=${PROJECT_INDEX_FILE:="index.php"} # change 'index.php' if it is required
+export PROJECT_DEV_INDEX_FILE=${PROJECT_DEV_INDEX_FILE:="index_dev.php"} # change 'index_dev.php' if it is required
```

Comment:
Since famous frameworks like Symfony and Laravel won't support multiple indexes we could remove it as well